Tests/ArmoniK.EndToEndTests/ArmoniK.EndToEndTests.Client/Tests/UnitTestHelperBase.cs

```diff
@@ -101,9 +101,8 @@ public void InitLogger()
                                         applicationNamespace,
                                         applicationService);
 
-    Props = new Properties(TaskOptions,
-                           Configuration.GetSection("Grpc")["EndPoint"],
-                           5001);
+    Props = new Properties(Configuration,
```

Comment:
This fix should be in another PR and why this constructor exists if we don't want it ?

Reply:
Historical reasons, we could remove it possibly but it may be used by the client iirc